### ISSUE TITLE
different from the paper

### ISSUE THREAD

**Issue by Baichenjia** [philipperemy/keras-tcn#45]
In `tcn/tcn.py`
```
def residual_block(x, dilation_rate, nb_filters, kernel_size, padding, dropout_rate=0):
    prev_x = x
    for k in range(2):
        x = Conv1D(filters=nb_filters,
                   kernel_size=kernel_size,
                   dilation_rate=dilation_rate,
                   padding=padding)(x)
        x = Activation('relu')(x)
        x = SpatialDropout1D(rate=dropout_rate)(x)

    x = Convolution1D(nb_filters, 1, padding='same')(x)
    res_x = keras.layers.add([prev_x, x])
    return res_x, x
```
I think the code ` x = Convolution1D(nb_filters, 1, padding='same')(x)` in the penultimate line 3 is not correct. This code want to match the shapes of `prev_x` and `x`. I think it's input should be `prev_x` and not `x`.

Change the last 3 lines to: 
```    
prev_x = Conv1D(nb_filters, 1, padding='same')(prev_x)
res_x = keras.layers.add([prev_x, x])
return res_x, x
` ``

**Comment by HighPerformance01** [comment]
Hi philipperemy

Maybe this is related to the little drop to my earlier question (can improve performance)?

Thank you

**Comment by philipperemy** [comment]
@Baichenjia great catch! Going to change it right now!
@HighPerformance01 will let you know ;)

**Comment by philipperemy** [comment]
Fixed https://github.com/philipperemy/keras-tcn/commit/81a68051ec40e0dde979fcfedf6461bdfb7f56c9

To get the latest version:
```
pip install keras-tcn --upgrade
```

**Comment by Baichenjia** [comment]
Thanks to the adoption of comments.

I have written a Github repository named "Tensorflow-TCN" (https://github.com/Baichenjia/Tensorflow-TCN), which is an implementation of *Temporal Convolutional Network* with tensorflow 1.13 (eager execution).

The explanation and graph in this README.md refers to [Keras-TCN](https://github.com/philipperemy/keras-tcn).

I have add links to [Keras-TCN](https://github.com/philipperemy/keras-tcn) in my README.md.

I will be very grateful if you can add a hyperlink to [Tensorflow-TCN](https://github.com/Baichenjia/Tensorflow-TCN) in README.md of [Keras-TCN](https://github.com/philipperemy/keras-tcn).

Thank you very much! @philipperemy
